pages/dkp/konvoy/2.0/quick-start-azure/index.md

```diff
@@ -6,7 +6,7 @@ excerpt: Get started by installing a cluster with default configuration settings
 draft: true
 beta: true
 enterprise: false
```

Comment:
I think we decided this should NOT be published (per Deepak last beta) @dkoshkin @msdolbey

Reply:
yeah, see comments on Azure https://jira.d2iq.com/browse/D2IQ-77180

Reply:
k, this looks right - draft:true.